libs/sdk-core/src/models.rs

```diff
@@ -387,9 +388,27 @@ impl TryFrom<i32> for ReverseSwapStatus {
     }
 }
 
-/// Trait covering functionality involving swaps
+/// Trait covering Breez Server reverse swap functionality
 #[tonic::async_trait]
 pub(crate) trait ReverseSwapperAPI: Send + Sync {
```

Comment:
Perhaps to better differentiate between the two similar named traits we can rename this to: `ReverseSwapperRoutingAPI` ?

Reply:
Done in f107829294d848f9b36f63af9ce713a593d3fdef